src/components/global-view.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import Grid from '@material-ui/core/Grid';
+import GlobalViewHeader from './global-view-header';
+import GlobalViewCore from './global-view-core';
+
+const GlobalView = ({ processName }) => {
+    const start = new Date();
+    start.setHours(0, 30, 0, 0);
+    var end = new Date();
```

Comment:
you can use const instead of var.

Reply:
the var word has been replaced by const word